_attributes/common-attributes.adoc

```diff
@@ -122,10 +122,10 @@ endif::[]
 :product-dedicated: Red Hat OpenShift Dedicated
 :SMProductName: Red Hat OpenShift Service Mesh
 :SMProductShortName: Service Mesh
-:SMProductVersion: 2.1.2.1
+:SMProductVersion: 2.1.3
 :MaistraVersion: 2.1
 //Service Mesh v1
-:SMProductVersion1x: 1.1.18.1
+:SMProductVersion1x: 1.1.18-2
```

Comment:
is it supposed to be `1.1.18.2` ?  I don't know the answer but  I see above that it was `2.1.2.1`.